projects/BFDR/IJENatality.cs

```diff
@@ -4937,12 +4956,43 @@ public string ATTEND_NPI
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: 
-                return "";
+                string fhirFieldName = "NPI";
+                string ijeFieldName = "NPI";
+                int npiLength = 10;
+                string npi = record.AttendantNPI;
+                if (!String.IsNullOrWhiteSpace(npi))
+                {
+                    string formattedNPI = npi.Replace("-", string.Empty).Replace(" ", string.Empty);
+                    if (formattedNPI.Length != npiLength)
+                    {
+                        validationErrors.Add($"Error: FHIR field {fhirFieldName} contains string '{npi}' which is not the expected length (without dashes or spaces) for IJE field {ijeFieldName} of length {npiLength}");
+                    }
+                    return Truncate(formattedNPI, npiLength).PadRight(npiLength, ' ');
+                }
+                else
+                {
+                    return new String(' ', npiLength);
+                }
             }
             set
             {
-                // TODO: Implement mapping to FHIR record location: 
+                string fhirFieldName = "NPI";
+                string ijeFieldName = "NPI";
+                int npiLength = 9;
+                if (!String.IsNullOrWhiteSpace(value))
+                {
+                    string npi = value.Trim();
+                    if (npi.Contains("-") || npi.Contains(" "))
+                    {
+                        validationErrors.Add($"Error: IJE field {ijeFieldName} contains string '{value}' which cannot contain ` ` or `-` characters for FHIR field {fhirFieldName}.");
+                    }
+                    string formattedNPI = npi.Replace("-", string.Empty).Replace(" ", string.Empty);
```

Comment:
this gets called in the FHIR setter too right? does it need to be in both places?